core/utils.py

```diff
@@ -31,6 +31,7 @@
 import time
 import unicodedata
 import urllib
+import urllib.request as urlrequest
```

Comment:
Remove this.

Reply:
Done

Reply:
Show python lint error, use `import urllib.request as urlrequest` to remove this.

Reply:
Can you screenshot the error?

Reply:
![Screenshot from 2021-12-01 18-30-26](https://user-images.githubusercontent.com/80846488/144239714-2b644963-c84c-4bcf-a236-62db6b369aca.png)

Reply:
Ah, okay. Please remove this check https://github.com/oppia/oppia/blob/develop/scripts/linters/general_purpose_linter.py#L211-L216 and then use `urllib.request` everywhere.

Reply:
Done